packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html

```diff
@@ -114,7 +114,7 @@
                                         ></span>
                                         <span
                                             data-ng-if="subitem.new"
-                                            class="oui-badge_beta oui-badge_new ml-2 oui-badge"
+                                            class="oui-badge oui-badge_s oui-badge_new ml-2"
```

Comment:
**suggestion:** Following file can be removed (I guess).

Path: `packages/manager/modules/pci/src/projects/project/sidebar/sidebar.scss`

Reply:
Done.